be/src/olap/cumulative_compaction_policy.cpp

```diff
@@ -84,7 +84,7 @@ void SizeBasedCumulativeCompactionPolicy::calculate_cumulative_point(
         bool is_delete = tablet->version_for_delete_predicate(rs->version());
 
         // break the loop if segments in this rowset is overlapping, or is a singleton.
```

Comment:
Please modify the comment after change the code~

Reply:
comment resolved

Reply:
> Please modify the comment after change the code~

comment resolved